src/components/containers/DashboardContainer.js

```diff
@@ -24,17 +24,15 @@ class DashboardContainer extends React.Component {
         if (dataFetch.pending) {
             return <Spinner />;
         } else if (dataFetch.rejected) {
-            const extraErrorComponentProps = {};
-
             if (dataFetch.reason && dataFetch.reason.message) {
-                extraErrorComponentProps.message = dataFetch.reason.message;
+                // eslint-disable-next-line no-console
+                console.error(dataFetch.reason.message);
```

Comment:
Yea I am totally fine with these even though the community has taken a dislike to console logs. If this was an internal project I'd even consider logging the dataFetch object below that for quicker inspection, though it might be much for something public facing.

Reply:
I didn't know the community took a dislike to these. Can you point me to some info about that?